### DISCUSSION TITLE
Suggestion: New App Icon?

### DISCUSSION THREAD

**Discussion by timolins** [joelekstrom/fastmate#109] [General]
Hi everyone! Thanks for making fastmate, I really like it.

The default icon didn't appeal to me, so I created my own. I thought I'll share if anyone else wants to use it – maybe even a PR if you want.

<img width="758" alt="Fastmate-preview" src="https://user-images.githubusercontent.com/1440854/187048585-0c0327cf-ade1-4ed5-b211-3406b7caf6fc.png">

<details>
  <summary>Full icon as PNG</summary>
 
<img width="1024" alt="fastmale-icon" src="https://user-images.githubusercontent.com/1440854/187048584-75a8169b-1164-4963-9faf-8cc68fd5e187.png">
</details>

**Comment by joelekstrom** [comment]
Looks great 😊 thanks for sharing! Might be time to open a discussion thread where we can keep an index of alternative icons since there have been a few.

I won’t change the default one (at least for now), partly because I like it and partly because this one looks a bit too close to the Fastmail icon for me to want to use it for legal reasons. But it’s great to have a set of alternative ones that people can choose from!

**Comment by timolins** [comment]
Great, a discussion thread sounds good.

Did some more variations, as I felt like I have too many white + blue circle apps in my dock.

<img width="1052" alt="Screenshot 2022-09-02 at 12 17 10" src="https://user-images.githubusercontent.com/1440854/188118488-dcdf125f-62d0-41d5-be78-ea5b8cf968ab.png">

Currently sporting the dark version, which I really like:
<img width="758" alt="Fastmate-preview" src="https://user-images.githubusercontent.com/1440854/188118647-b4b527cf-0579-43dc-9ca6-00e93aafc2d6.png">

**Comment by YANOUSHek** [comment]
I love this alternative. Feels more inline with other macOS icons.

@joelekstrom I understand your decision to not change the icon, but maybe it would be cool to embed more than one icon in to the app and allow users to change them in the settings? That way when the app updates it would keep the user selected icon instead of reverting to the default one and waiting for me to "override" the icon again. WDYT? I think I can manage a PR with that if it's OK with you.

**Comment by YANOUSHek** [comment]
@timolins could you share the other color variants in full resolution?

**Comment by timolins** [comment]
Sure! I picked the five versions I liked best.

<details>
  <summary>Show PNGs</summary>
 
<img width="1024" alt="fastmate-1" src="https://user-images.githubusercontent.com/1440854/188452648-b1297588-ac73-43e8-8fb4-1ba8c1d087c0.png">
<img width="1024" alt="fastmate-2" src="https://user-images.githubusercontent.com/1440854/188452686-72ac3671-3f31-4563-8b35-2b0bd816b2ba.png">
<img width="1024" alt="fastmate-3" src="https://user-images.githubusercontent.com/1440854/188452709-4068ea33-c8b4-4197-ae4e-93068f08694a.png">
<img width="1024" alt="fastmate-4" src="https://user-images.githubusercontent.com/1440854/188452736-babb6c4b-afd0-4e52-80ec-e57451ffa8c7.png">
<img width="1024" alt="fastmate-5" src="https://user-images.githubusercontent.com/1440854/188452763-17ff6b65-96a8-441f-b5c0-78c1500ea153.png">

</details>

**Comment by melvinsalas** [comment]
I am the designer of the current icon and I agree that the change can be made :3

**Comment by mikescott** [comment]
I love the first icon.

If anyone wants to use it you can easily change an app icon by first copying the PNG to your clipboard, then right clicking on Fastmate.app, and then "Get Info". In the new window click on the tiny icon in the top left (it should get outlined in blue) and do CMD+V to replace the icon. The only downside to this is you'll need to do it after every app update.

**Comment by frankievalentine** [comment]
@timolins Beautiful icons! Thanks for those. Would you share your figma file for those by chance? No big deal if not.

@mikescott Use [Pictogram](https://pictogramapp.com/) to have your icons persist after updates.

**Comment by timolins** [comment]
@frankievalentine For sure, here is the Figma: https://www.figma.com/file/bCeqIyW5py70jle8Q1H1Dy/Fastmate-Icons
